benchexec/tablegenerator/__init__.py

```diff
@@ -1245,7 +1249,14 @@ def create_tables(
     # get common folder of sourcefiles
     # os.path.commonprefix can return a partial path component (does not truncate on /)
     common_prefix = os.path.commonprefix([r.id.name for r in rows])
-    common_prefix = common_prefix[: common_prefix.rfind("/") + 1]
+    separator = (
```

Comment:
There is `os.sep` on Python, which already provides the separator. So we would just need to check for URLs (which could also be different from HTTP, I think we usually check for `://` to detect URLs).